node/Cargo.toml

```diff
@@ -84,7 +85,7 @@ sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate", bra
 sp-authority-discovery = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19" }
 
 #Primitives
-polkadex-primitives = { git = "https://github.com/Polkadex-Substrate/polkadex-primitives.git", branch = 'polkadot-v0.9.19'}
+polkadex-primitives = { git = "https://github.com/Polkadex-Substrate/polkadex-primitives.git", branch = "fx-polkadot-v0.9.19", default-features = false }
```

Comment:
Why this is referring this branch? I think you need to change to  `polkadot-v0.9.19`